tasks/galera.yaml

```
@@ -0,0 +1,58 @@
+---
+- name: Check if grastate.dat exists
+  stat:
```

Task is not idempotent but read-only so we should use `changed_when: false`.

```suggestion
  changed_when: false
  stat:
```